ibctest/test_setup.go

```diff
@@ -33,8 +33,13 @@ func SetupTestRun(t *testing.T) (context.Context, string, *dockertest.Pool, stri
 	if err != nil {
 		return ctx, "", nil, "", err
 	}
+
+	// schedule cleanup for afterwards
 	t.Cleanup(dockerCleanup(t.Name(), pool))
 
+	// run cleanup for this test first in case previous run was killed (e.g. Ctrl+C)
+	dockerCleanup(t.Name(), pool)()
```

Comment:
I'm unsure the implications here but worth a shot. Easy enough to revert.

Reply:
Just means if you killed a running test with Ctrl+C, so cleanup does not happen, then run the test again, it will run the cleanup before starting so that you don't run into `container already exists` errors.